gazelle/js/typescript/tsconfig.go

```diff
@@ -183,12 +214,19 @@ func (c TsConfig) ExpandPaths(from, p string) []string {
 			matchedText := p[len(m.prefix) : len(p)-len(m.suffix)]
 			mappedPath := strings.Replace(originalPath, "*", matchedText, 1)
 
-			mappedPath = path.Clean(mappedPath)
```

Comment:
I guess if tests pass this wasn't needed? 🤔 

All the other `possible` paths get cleaned though so maybe we should do it (but after the `Join` below).

Reply:
Join should do a clean anyways don't it?

Reply:
I'm not sure. If it does we should remove all the `Clean`s in a followup commit.

Reply:
> Join joins any number of path elements into a single path, separating them with slashes. Empty elements are ignored. The result is Cleaned. ~ https://pkg.go.dev/path#Join with https://cs.opensource.google/go/go/+/master:src/path/path.go;l=179

So the Clean is performed by Join already.